src/lib/provable/test/bitwise.unit-test.ts

```diff
@@ -183,6 +193,18 @@ await equivalentAsync({ from: [maybeField, maybeField], to: field }, { runs })(
   }
 );
 
+await equivalentAsync({ from: [maybeField, maybeField], to: field }, { runs })(
+  (x, y) => {
+    if (x >= 2n ** 64n || y >= 2n ** 64n)
+      throw Error('Does not fit into 64 bits');
+    return x | y;
+  },
+  async (x, y) => {
+    let proof = await Bitwise.or(x, y);
+    return proof.publicOutput;
+  }
+);
+
```

Comment:
I'd say remove this, the in-circuit equivalence tests above are much more thorough and creating a proof here just makes CI slower

Reply:
removed!

Reply:
Huh, I meant just this test but you removed all of them. I think it's not a bad idea to remove most of them, but can you at least keep 1-2 of them?

Reply:
Which ones should we keep?

Reply:
Let's keep the first two